src/DecryptionFailureTracker.ts

```diff
@@ -129,9 +129,13 @@ export class DecryptionFailureTracker {
     //     localStorage.setItem('mx-decryption-failure-event-ids', JSON.stringify([...this.trackedEvents]));
     // }
 
-    public eventDecrypted(e: MatrixEvent, err: MatrixError): void {
+    public eventDecrypted(e: MatrixEvent, err: DecryptionError): void {
+        // for now we only track megolm decrytion failures
+        if (e.event.content?.algorithm != "m.megolm.v1.aes-sha2") {
```

Comment:
nitpick: this should use the getContent() method
```suggestion
        if (e.getContent()?.algorithm != "m.megolm.v1.aes-sha2") {
```

Reply:
Thx, I will use getWireContent() because I want the encrypted content (although here as it's a decryption failure it won't make a difference, but seems more correct)